spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/list.html

```diff
@@ -37,9 +119,7 @@
                     },
```

Comment:
Please try to put a different duration for hide, so that it doesn't look weird to the user that the page refreshes while the dialog is being close